src/game/Damage.cpp

```diff
@@ -1402,7 +1403,8 @@ void ARX_DAMAGES_DurabilityRestore(Entity * io, float percent) {
 			
 			float mloss = io->max_durability * v;
 			if(io->ioflags & IO_ITEM) {
-				io->_itemdata->price -= static_cast<long>(float(io->_itemdata->price) * v);
+				io->_itemdata->buyPrice -= util::to<long>(float(io->_itemdata->buyPrice) * v);
```

Comment:
can it happen here and in 1392 that we get a negative price?

Reply:
No idea, I've just renamed the property, but I'll check it.

Reply:
very nice writeup 👍

Reply:
nope thats it. ready to merge